README.md

```diff
@@ -66,6 +66,7 @@ If you want to leave the firmware image, manifest and campaign to your account a
 New manifest-tool 2.0.0 supports two different versions of manifest schemas. Update test's manifest creation defaults to 'v1' manifest version, but you can set the version e.g to 'v3' by the `--manifest_version=v3` startup argument.
 Supported versions at the moment are 'v1' and 'v3'.
 
+If you need to pass in an explicit firmware version, verses using a time-stamp-generated value, use the `--fw_version` option.
```

Comment:
verses -> versus?

Reply:
Nice catch. Your English skills are much better than mine and I'm a native speaker!

Reply:
I just made a commit to fix the typo